docs/modules/ROOT/pages/contributing/developers.adoc

```diff
@@ -109,6 +109,24 @@ After you log in into your cluster, you can run the following command to execute
 make test-integration
 ----
 
+[NOTE]
```

Comment:
There is a page dedicated to e2e testing, that may be a better location for documenting this: https://camel.apache.org/camel-k/latest/contributing/e2e.html.

Reply:
Yeah but the existing e2e doc doesn't talk about `make test-integration` there. At this moment, doc on e2e testing is split into two places. I agree that we need to consolidate the doc and e2e.adoc is the right place.